src/utils/constants.ts

```diff
@@ -124,6 +127,7 @@ export const PERMIT2: Record<number, Address> = {
     [ChainId.SONIC]: '0x000000000022D473030F116dDEE9F6B43aC78BA3',
     [ChainId.ARBITRUM_ONE]: '0x000000000022D473030F116dDEE9F6B43aC78BA3',
     [ChainId.BASE]: '0x000000000022D473030F116dDEE9F6B43aC78BA3',
+    [ChainId.MONAD_TESTNET]: '0x000000000022D473030F116dDEE9F6B43aC78BA3',
```

Comment:
I don't think this address is correct for monad testnet

https://testnet.monadexplorer.com/address/0x000000000022D473030F116dDEE9F6B43aC78BA3?tab=Activity